lib/tag-field.jsx

```diff
@@ -131,7 +135,7 @@ export default React.createClass( {
 					onKeyDown={this.onKeyDown}
 					onBlur={this.onBlur}>
 					<input className="hidden-tag" tabIndex="-1" ref="hiddenTag" onKeyDown={this.preventTyping} />
-					{this.props.tags.map( ( tag, index ) =>
+					{ uniq( tags ).map( ( tag, index ) =>
```

Comment:
shouldn't this happen when the tags are loaded into state, otherwise you'll end up needing to use `uniq` everywhere you render tags, which kind of defeats the purpose of having a central place to store state